aiohttp/client.py

```diff
@@ -639,6 +620,27 @@ def ws_connect(
             max_msg_size: int=4*1024*1024
     ) -> ClientWebSocketResponse:
 
+        ws_timeout: ClientTimeout
```

Comment:
This is not Python 3.5 friendly: https://travis-ci.com/aio-libs/aiohttp/jobs/220023040#L511

Reply:
ok thanks!